JS.js

```diff
@@ -5,14 +5,19 @@ let highscore = 0;
 const displayMessage = function (message) {
   document.querySelector('.message').textContent = message;
 };
+
+
 document.querySelector('.check').addEventListener('click', function () {
   const guess = Number(document.querySelector('.guess').value);
-  console.log(guess, typeof guess);
+
```

Comment:
you have to check if the number is between 1 and 20 and alert the user if he write a number out of this range